src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -198,13 +198,19 @@ def to_ir(self, workflow_def: WorkflowDef) -> WorkflowRun:
         )
 
 
-class RuntimeType(str, Enum):
+class Resources(pydantic.BaseModel):
     """
     Implements:
-    https://github.com/zapatacomputing/workflow-driver/blob/2b3534/openapi/src/schemas/Runtime.yaml
+        https://github.com/zapatacomputing/workflow-driver/blob/6270a214fff40f53d7b25ec967f2e7875eb296e3/openapi/src/schemas/Resources.yaml
     """
 
-    SINGLE_NODE_RAY_RUNTIME = "singleNodeRayRuntime"
+    cpu: Optional[str] = pydantic.Field(
+        regex=r"^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$"
```

Comment:
Nonblocking: Oh noes, a long and complex regex. Could you somehow maybe document what it means? Same for GPU regex

Reply:
The regexes are from the spec: https://github.com/zapatacomputing/workflow-driver/blob/6270a214fff40f53d7b25ec967f2e7875eb296e3/openapi/src/schemas/Resources.yaml

This is in the docstring.